configure.ac

```diff
@@ -79,9 +79,8 @@ AC_CHECK_HEADERS([sys/cdefs.h sys/file.h sys/mman.h sys/param.h \
 AC_CHECK_HEADERS([bzlib.h],
                  [],
                  [AC_MSG_FAILURE([missing <bzlib.h>; is bzip2 installed?])])
-AC_CHECK_HEADERS([json.h],,
```

Comment:
Thanks for the review. 
Well, i'm not sure. If we use AC_CHECK_HEADER, it means we have to manually tinker with CFLAGS if json-c is installed on a non-standard location, while the main purpose of this change is to avoid that. Anyway, i push a commit with this and we can review